src/pages/auth/Login.tsx

```diff
@@ -293,6 +316,7 @@ export default function Login() {
                 )}
 
                 <PopupModal
+                  type={modalContent.type}
```

Comment:
Send all the popup modal content as children, without making it dependent on the type.